cpp/tests/utilities/column_utilities.cu

```diff
@@ -343,6 +343,10 @@ struct column_view_printer {
       out = std::move(h_data.first);
     }
   }
+
+  template <typename Element, typename std::enable_if_t<std::is_same<Element, cudf::dictionary32>::value>* = nullptr>
+  void operator()(cudf::column_view const& col, std::vector<std::string> & out) {
```

Comment:
So is printing dictionaries not supported yet? Perhaps this should be a CUDF_FAIL?